tools/export.py

```diff
@@ -204,6 +207,14 @@ def main(args):
     fake_data = get_fake_input(cfg, device=device)
 
     # BEGIN nncf part
+    alt_ssd_export = getattr(args, 'alt_ssd_export', False)
```

Comment:
```suggestion
    alt_ssd_export = args.alt_ssd_export
```
??

Reply:
No, I think we cannot do that. The reason is that we have two subparsers, for `openvino` and for `onnx`, and `onnx` subparser does not contain `alt_ssd_export` parameter.

Please, see below in the function `parse_args`

But this code is called both for `openvino` and `onnx` cases.

Reply:
Ok, I see. This way 
```suggestion
    alt_ssd_export = args.target == 'openvino' and args.alt_ssd_export
```
looks more explicit and transparent.

Reply:
Pavel, note that I just copied the code `getattr(args, 'alt_ssd_export', False)` from several lines below. 
And I think that just check if `alt_ssd_export` is in attributes is more pythonic here...

But if you relly insist on this change, I will apply it

Reply:
Guess who is the author of the code which is several lines below 😉 I have no will to insist anymore. You may leave it as is, if you wish.

Reply:
Thank you